packages/terra-docs/src/Button.module.scss

```diff
@@ -0,0 +1,43 @@
+@import './clinical-lowlight-theme/Button.module';
+@import './orion-fusion-theme/Button.module';
+
+/* stylelint-disable terra/custom-property-namespace */
```

Comment:
this doesn't support scoped packages see issue: https://github.com/cerner/stylelint-config-terra/issues/44

Reply:
it does now